src/node/setupServer.ts

```diff
@@ -19,10 +19,12 @@ class SetupServerApi extends BaseSetupServerApi implements SetupServer {
    */
   protected override onRequest(request: Request): void {
     try {
-      setMaxListeners(
-        Math.max(defaultMaxListeners, this.currentHandlers.length),
-        request.signal,
-      )
+      if (typeof setMaxListeners === 'function') {
```

Comment:
we need this check in jsdom with the mjs file. We don't with cjs, apparently.  Not sure why that is offhand